definitions/npm/@segment/analytics_v3.11.2/flow_v0.103.x-/test_analytics_v3.11.2.js

```diff
@@ -0,0 +1,19 @@
+window.analytics.load("apiKey");
```

Comment:
It might work if you just kind of hack the situation? Something like...

`const window: { analytics: MyType } = { analytics: ({}: any) }`

If you wrap the tests in a `describe` then the statements are never actually run in a JS context so that might work out.

Give that a shot?